api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -34,6 +34,7 @@
   private String freeTierBillingProjectName;
   private Timestamp firstSignInTime;
   private Set<Authority> authorities = new HashSet<Authority>();
+  private Set<WorkspaceUserRole> workspaces = new HashSet<WorkspaceUserRole>();
```

Comment:
It's a little weird to call this "workspaces"... rename to workspaceUserRoles? Or workspacesWithRoles?